docs/Makefile

```diff
@@ -19,8 +19,17 @@ help:
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
 
-full-html: html rust-html
+python-and-rust-html: html rust-html
 	true  # need a non-empty rule to prevent matching the %: rule.
 
 rust-html:
-	cargo doc --no-deps --workspace --all-features --target-dir $(BUILDDIR)/html/rust
+	RUSTDOCFLAGS="--enable-index-page -Z unstable-options" cargo doc \
+		--no-deps \
+		--workspace \
+		--exclude bench-vortex \
+		--exclude notimplemented \
+		--exclude tpch-benchmark \
```

Comment:
you only need `--exclude bench-vortex` and `--exclude xtask`, the other two are just binary targets inside of bench-vortex

Reply:
ah

Reply:
done.